tuna/corrupt_configs.py

```diff
@@ -47,31 +47,51 @@ def get_logger():
   return logger
 
 
-def main():
+LOGGER = get_logger()
+
+
+# pylint: disable-msg=too-many-locals
+def main() -> None:
   """Main module function"""
-  file_name = None
-  arch = None
+  file_name: Optional[str] = None
+  arch: Optional[str] = None
+
   if len(sys.argv) == 3:
     file_name = sys.argv[1]
     arch = sys.argv[2]
   else:
     sys.exit("Usage: " + sys.argv[0] + " <output_file> <arch>")
 
-  table_cols_conv_invmap = {v: k for k, v in TABLE_COLS_CONV_MAP.items()}
-  table_cols_fusion_invmap = {v: k for k, v in TABLE_COLS_FUSION_MAP.items()}
+  tuple_idx: Dict[Any, Any]
+  key_idx: Dict[Any, Any]
+  table_cols_conv_invmap: dict = {
+      tuple_idx: key_idx for key_idx, tuple_idx in TABLE_COLS_CONV_MAP.items()
+  }
+  table_cols_fusion_invmap: dict = {
+      tuple_idx: key_idx
+      for key_idx, tuple_idx in TABLE_COLS_FUSION_MAP.items()
+  }
 
+  outfile: TextIOWrapper
+  cur: mysql.connector.cursor.MySQLCursor
   with open(file_name, "w") as outfile:  # pylint: disable=unspecified-encoding
-    sub_cmd_idx = None
+    sub_cmd_idx: int = 0
     with DbCursor() as cur:
       cur.execute(
-          "select config.* from job inner join config on config.id = job.config where \
-              job.arch = %s and reason = 'corrupt';", (arch,))
+          "select conv_config.* from job inner join conv_config on conv_config.id = \
+          job.conv_config  where job.arch = %s and reason = 'corrupt';",
+          (arch,))
       sub_cmd_idx = cur.column_names.index('cmd')
-      count = 0
+      count: int = 0
 
+      row: List = []
+      sub_cmd: str
```

Comment:
Possible to move these hints to the location of their previous declarations?
sub_cmd is at 90 within the loop scope.
Not sure if you can use type hints within loop definitions.
Looks like setting empty list ( = [] ) wouldn't be necessary. There are no values for type hints on lines 92 - 94.

Reply:
moved declarations of count,row,sub_cmd to before using with, for loop, 92-94 idx,fds,arg_name are initalized to 0 and ""